### DISCUSSION TITLE
Languages name in hide param

### DISCUSSION THREAD

**Discussion by rb58853** [anuraghazra/github-readme-stats#3837] [Q&A] [answered]
How can i know the name of a language? For example, i dont know how put Jupyter Notebook! 
I was try with:

"...&hide=jupyternotebook,jupyter,notebook,ipynb,jupiter_notebook,JupiterNotebook..."

The hell, nothing working😑. 

Please, any idea???

**Comment by qwerty541** [comment, marked as answer]
Hey @rb58853. Our documentation contains the notice which clearly says that languages should be URL encoded.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/1e62d4b6-cd84-460d-b472-8f4220cd14d1)

So the correct URL will be:

```html
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=rb58853&layout=donut&theme=dark&border_color=red&hide=jupyter%20notebook" />
```

<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=rb58853&layout=donut&theme=dark&border_color=red&hide=jupyter%20notebook" />
